src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -2540,6 +2543,16 @@ ExitInfo ExecutorWorker::handleOpsFileAccessError(SyncOpPtr syncOp, ExitInfo ops
     return removeDependentOps(syncOp);
 }
 
+ExitInfo ExecutorWorker::handleOpsFileNotFound(SyncOpPtr syncOp, ExitInfo opsExitInfo) {
+    if (syncOp->targetSide() != ReplicaSide::Remote) {
+        LOGW_SYNCPAL_WARN(_logger, L"Invalid target side for SystemError | FileNotFound error: " << syncOp->targetSide());
```

Comment:
```suggestion
        LOGW_SYNCPAL_WARN(_logger, L"Invalid target side for ExitInfo(SystemError, FileNotFound error): " << syncOp->targetSide());
```
We may also write a `toString` function for `ExitInfo`.

Reply:
It was not as trivial as I first thought, but here it is 👍🏾 : [f288be0](https://github.com/Infomaniak/desktop-kDrive/pull/396/commits/f288be06968876e6bfcf2eedfef10c68e88dd0d1).